### PR TITLE
Add initial README.md file for Travel Tracker App

### LINKED ISSUE #8: Add professional README
## Description:

A high quality README is essential for all software projects, in order for new developers to get up to speed and contribute quickly.  For hobby projects, they are useful to show off the work done to your employers.  

Attached below is an example of a README.md file from a project I worked on, the raw markdown text

https://raw.githubusercontent.com/lpile/calorie-counts/master/README.md

and here it is as it will show up on github:
https://github.com/lpile/calorie-counts/blob/master/README.md

I suggest copy pasting the exact raw text from mine, and fixing it up from there. (remove testing, instead of talking about endpoints we will add some screenshots and descriptions of features...this can be blank for now)

gist.github.com is a great place to write your rough draft and see how it looks as markdown, before opening the pull request. 

## Checks:
- [ ] implemented descriptive and easy to read markdown changes to README.md

### PR DESCRIPTION
**Pull request by josh-everett01**
## Changes proposed by this PR
closes #8 

## What did you struggle on to complete?
Making sure formatting was correct and also making sure I was adding the correct information

## Checklist:
- [x] code has been linted with ESLint
- [x] I have reviewed my code
- [x] all issue criteria is completed 
- [x] I have fully styled all changes

## Helpful Resources:
[link_1](https://example_resource.com)
[link_2](https://example_resource_2.com)

### CODE REVIEW

File: src/README.md [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+$ npm install
+```
+### Production Links
+###### [Frontend Interface](https://github.com/josh-everett01/travel_tracker)
+
+### Versions
+ 
+`npm 6.14.7`  
+
+### Schema
```
Comment (Mycobee) [inline src/README.md:59]: remove

File: src/README.md [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+
+### Tech Stack
+* JavaScript
+* HTML5/CSS3/SCSS
+
+### Project Boards
+###### [Travel Tracker Project Board](https://github.com/josh-everett01/travel_tracker/projects/1)
+
+### Endpoints
```
Comment (Mycobee) [inline src/README.md:68]: Remove this whole section (all endpoints)

File: src/README.md [hunk 3]
```diff
@@ -0,0 +1,220 @@
+## Travel Tracker
+
+### Overview
+Travel Tracker is an application which manages and tracks different trips for
```
Comment (Mycobee) [inline src/README.md:4]: Say something like 
Travel tracker is an app that manages and tracks different trips for users and travel agents, by consuming and posting to an external API

File: src/README.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+for that user - Delete an upcoming trip
+for that user
+
+### Setup
+
+**Git Setup**
+
+```
+$ git clone git@github.com:josh-everett01/travel_tracker.git
+$ cd calorie_counts
```
Comment (Mycobee) [inline src/README.md:38]: `cd calorie_counts`? :)

File: src/README.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+cost (including a 10% travel agent fee) for the trip. - Once they submit the trip request, it will show on
+their dashboard as “pending” so that the travel agency
+can approve or deny it.
+An agent who is logged in is able to: - See and approve / deny trip requests - Search for any user by name and: - View their name, a list
+of all of their trips,
+and the total amount
+they’ve spent (including
+10% agent cut) - Approve a trip request
+for that user - Delete an upcoming trip
+for that user
```
Comment (Mycobee) [inline src/README.md:30]: Add some markdown formatting for these lines, perhaps h4s and bulleted lists?
https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet

**Review by Mycobee — CHANGES_REQUESTED** [review]
Looks good @josh-everett01 

A couple very small changes and this is good to merge.

**Review by josh-everett01 — COMMENTED** [review]
complete

**Review by Mycobee — APPROVED** [review]

**Pull request merged**